packages/server/src/sdk/app/automations/crud.ts

```diff
@@ -116,8 +124,7 @@ export async function create(automation: Automation) {
 }
 
 export async function update(automation: Automation) {
-  automation = { ...automation }
-
+  automation = trimUnexpectedObjectFields({ ...automation })
```

Comment:
Do we need these spreads? Does this not result in two copies of the object when we only need one?